app/services/form1010cg/service.rb

```diff
@@ -32,14 +34,56 @@ def initialize(claim)
     #
     # @return [Form1010cg::Submission]
     def process_claim!
+      raise 'submission already present' if submission.present?
+
       assert_veteran_status
 
       carma_submission = CARMA::Models::Submission.from_claim(claim, build_metadata).submit!
 
-      Form1010cg::Submission.new(
+      @submission = Form1010cg::Submission.new(
         carma_case_id: carma_submission.carma_case_id,
         submitted_at: carma_submission.submitted_at
       )
+
+      submit_attachment
+
+      submission
+    end
+
+    # Will generate a PDF version of the submission and attach it to the CARMA Case.
+    #
+    # @return [Boolean]
+    def submit_attachment # rubocop:disable Metrics/MethodLength
+      raise 'requires a processed submission'     if  submission&.carma_case_id.blank?
+      raise 'submission already has attachments'  if  submission.attachments.any?
+
+      file_path = begin
+                    claim.to_pdf
+                  rescue
+                    return false
+                  end
+
+      begin
+        carma_attachments = CARMA::Models::Attachments.new(
+          submission.carma_case_id,
+          claim.veteran_data['fullName']['first'],
+          claim.veteran_data['fullName']['last']
+        )
+
+        carma_attachments.add(CARMA::Models::Attachment::DOCUMENT_TYPES['10-10CG'], file_path)
+
+        carma_attachments.submit!
+        submission.attachments = carma_attachments.to_h
+      rescue
+        # Regardless of the reason, we should not raise an error when sending attachments fails.
+        # If we made it this far, there is a submission that exists in CARMA, so the user should
+        # get a sucessful response whether attachments reach CARMA or not.
```

Comment:
I guess i'm not following why one wouldn't care if the attachments don't successfully submit to CARMA

Reply:
The PDF we're sending to CARMA is just the submission, itself, as a PDF. This is to follow the current conventions of CARMA: every case has the PDF of the submission attached. The people who process the claims prefer to have the PDF as a reference.

So the end-user doesn't know an attachment is being sent with the submission at all. We also don't want to have an error bubble up to the response, because that will mislead the user to thinking the submission failed when it didn't.

I can update the code comment to be more descriptive of the business requirement here.

Reply:
Ok, makes sense now. An updated comment would be great!